samples/namespacescope-sample/createUIDefinition.json

```diff
@@ -0,0 +1,247 @@
+									"validationMessage": "Only alphanumeric characters are allowed, and the value must be 6-30 characters long."
+								}
+							},
+							{
+								"name": "kubernetesVersion",
+								"type": "Microsoft.Common.DropDown",
+								"label": "Kubernetes version",
+								"toolTip": "The version of Kubernetes that should be used for this cluster. You will be able to upgrade this version after creating the cluster.",
+								"constraints": {
+									"allowedValues": "[map(steps('clusterDetails').newClusterSection.aksVersionLookupControl.properties.orchestrators, (item) => parse(concat('{\"label\":\"', item.orchestratorVersion, '\",\"value\":\"', item.orchestratorVersion, '\"}')))]",
+									"required": true
+								}
+							},
+							{
+								"name": "vmSize",
+								"type": "Microsoft.Compute.SizeSelector",
+								"label": "VM size",
+								"toolTip": "The size of virtual machine for VM.",
+								"recommendedSizes": [
+									"Standard_B4ms",
+									"Standard_DS2_v2",
+									"Standard_D4s_v3"
+								],
+								"constraints": {
+									"allowedSizes": [
+										"Standard_B4ms",
+										"Standard_DS2_v2",
+										"Standard_D4s_v3"
+									],
+									"excludedSizes": []
+								},
+								"osPlatform": "Linux"
+							},
+							{
+								"name": "enableAutoScaling",
+								"type": "Microsoft.Common.CheckBox",
+								"label": "Enable auto scaling",
+								"toolTip": "Enable auto scaling",
+								"defaultValue": true						
+							},
+							{
+								"name": "vmCount",
+								"type": "Microsoft.Common.Slider",
+								"min": 1,
+								"max": 10,
+								"label": "VMCount",
+								"subLabel": "",
+								"defaultValue": 1,
+								"showStepMarkers": false,
+								"toolTip": "Specify VM count",
+								"constraints": {
+									"required": false
+								},
+								"visible": true
+							}
+						],
+						"visible": "[basics('createNewCluster')]"
+					}
+				]
+			},
+			{
+				"name": "extensionDetails",
+				"label": "Cluster Extension Details",
+				"elements": [
+					{
+						"name": "extensionResourceName",
+						"type": "Microsoft.Common.TextBox",
+						"label": "Cluster extension resource name",
+						"defaultValue": "",
+						"toolTip": "Only lowercase alphanumeric characters are allowed, and the value must be 6-30 characters long.",
+						"constraints": {
+							"required": true,
+							"regex": "^[a-z0-9]{6,30}$",
+							"validationMessage": "Only lowercase alphanumeric characters are allowed, and the value must be 6-30 characters long."
+						},
+						"visible": true
+					},
+					{
+						"name": "targetNamespace",
+						"type": "Microsoft.Common.TextBox",
+						"label": "Installation namespace",
+						"defaultValue": "default",
+						"toolTip": "The Kubernetes namespace in which the extension will be installed. If the namespace does not exist, it will be created. Extension resource name is used as default value",
```

Comment:
>Extension resource name is used as default value [](http://example.com/codeflow?start=138&length=48)

Is this statement still valid? I see we are passing 'default' as defaultValue in above line. 
Can user not enter anything in this textbox? If yes, would extension resource name be used?